### PR TITLE
feat: use Group to configure common blockchain permissions to users

### PR DESCRIPTION
**Pull request by bjwswang**
Fix: #84 
Fix: #100

### CODE REVIEW

File: config/samples/users/org2admin.yaml [hunk 1]
```diff
@@ -7,26 +7,7 @@ metadata:
...
-  groups:
-  - observability
-  - system:nodes
-  - system:masters
-  - resource-reader
-  - iam.tenxcloud.com
-  - observability
   name: org2admin
-  password: $2a$10$693K.zP98yCs1qVwEp//DuWYOtLIE1doihtGhcCyYh3IpgSdGGba2
+  password: passw0rd
```
Comment (0xff-dev) [inline config/samples/users/org2admin.yaml:11]: The password should be encrypted.

Comment (Abirdcfly) [reply config/samples/users/org2admin.yaml:11]: passw0rd -> $2a$10$eXHv4JD1hEfMA/vV6Whjae84bVI0CMeZpa43z97PurPQMmD8OAfe2

**Comment by bjwswang** [comment]
Example test failed due to we change the way authorizing blockchain common permission to users which requries changes in `installer` too.

**Review by 0xff-dev — APPROVED** [review]

**Review by Abirdcfly — CHANGES_REQUESTED** [review]

File: config/samples/example-test.sh [hunk 2]
```diff
@@ -234,7 +234,7 @@ sed -i -e "s/<org3AdminToken>/${Admin3Token}/g" config/samples/orgs/org3.yaml
 info "4.1.1 create org=org1, wait for the relevant components to start up."
 kubectl create -f config/samples/orgs/org1.yaml --dry-run=client -o json |
 	jq '.spec.caSpec.ingress.class = "'$IngressClassName'"' | jq '.spec.caSpec.storage.ca.class = "'$StorageClassName'"' |
-	kubectl create --as=org1admin -f -
+	kubectl create --as=org1admin --as-group=bestchains -f -
```
[the remark this answers is not in the document]
Comment (bjwswang) [reply config/samples/example-test.sh:237]: right…at least we should not use `--as-group`

[the remark this answers is not in the document]
Comment (Abirdcfly) [reply config/samples/example-test.sh:237]: https://github.com/bestchains/fabric-operator/issues/100#issuecomment-1425060800

**Review by Abirdcfly — APPROVED** [review]

**Pull request merged**